plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php

```diff
@@ -268,9 +268,9 @@ function createProductNode($doc, $submission, $publicationFormat) {
 			$titleElementNode->appendChild($this->_buildTextNode($doc, 'TitleWithoutPrefix', $publication->getLocalizedTitle()));
 		}
 
-		if ($publication->getData('subtitle')) {
-			$titleElementNode->appendChild($this->_buildTextNode($doc, 'Subtitle', $publication->getData('subtitle')));
-		}
+		// if ($publication->getData('subtitle')) {
+		// 	$titleElementNode->appendChild($this->_buildTextNode($doc, 'Subtitle', $publication->getData('subtitle')));
+		// }
```

Comment:
Is this commented for a reason?

Reply:
`$publication->getData('subtitle')` is an array (localized), and `_buildTextNode` expects string. I was looking for alternatives here

Reply:
ONIX is not multilingual-capable, so I would suggest using the values for the submission's locale. (Generally we should also be avoiding `getLocalizedData` because it gets the content based on the current user's locale -- exports should not depend on the user's language, so `getData('subtitle', $publication->getData('locale'))` is preferred.)